Engine/CMakeLists.txt

```diff
@@ -71,7 +69,18 @@ file(GLOB SOURCES
     "**/**/**/**/*.cpp"
   )
 
-add_library( ${PROJECT_NAME} SHARED ${SOURCES} ${AlcBackend} )
+# FIXME: the headers include some SOURCES files so installing them is useless
+# the files in include should be the headers themselves!
+file(GLOB PUB_HEADERS
+    "include/Tempest/**"
+    )
+
+add_library(${PROJECT_NAME} ${SOURCES} ${AlcBackend})
```

Comment:
Shared build not working: BUILD_SHARED_LIBS - not used.

Reply:
Sorry, I don't understand. Do you get a cmake error?

Reply:
> Do you get a cmake error?
It's not possible to build shared version with your change - every build produces static library.

Reply:
That doesn't seem right. Are you sure you're using cmake correctly? Have you tried a clean build directory?

Reply:
Answer is 'yes' and 'yes'.
Your script is incorrect, at line 78

Reply:
I don't see how. The macro use seems correct according to the docs.

Reply:
I've read the doc and made yet another rebuild with `ON` - I was wrong about it, sorry.
Anyway, because usage of BUILD_SHARED_LIBS not desirable.